travis.sh

```diff
@@ -4,6 +4,11 @@ set -ev
 
 ./liftsh test
 
+cd web/webkit
+npm run lint
+npm run test
+cd -
```

Comment:
Since `set -e` is used in the script it should stop and exit on the first non-zero return code.